src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md

```diff
@@ -84,17 +86,133 @@ Follow this process:
+
+* **Impact Assessment:** Assess the impact of the increased quota on your project's performance, scalability, and budget.
+
+If you need help answering this question [check these examples](#examples-of-answers).
+
+#### 3. What steps have you taken to try to fit your application into your current quota?
+
+Think about if there's anything you can do to use your current quota more efficiently. Can you decrease the resources assigned to your existing pods to free up more quota? 
+
+Write down what you've already done to lower current resource usage and explain why you need the quota increase. Be honest - if you've tried everything and still need more quota, that's okay. Sometimes, there's nothing you can reasonably do to make your existing quota work. If you can, explain why you think you shouldn't change anything.
+
+This part might include references to pods or objects you haven't mentioned before. Even if you're asking for more quota to run more frontend application pods, you might find that your database pods are using up more resources than they need. Explain why you can't reduce the resource allocation of these other pods, or try doing that before asking for more quota.
+
+For more information on quotas, see [OpenShift project resource quotas](../automation-and-resiliency/openshift-project-resource-quotas.md).
+
+Before we move forward with your quota change request, expect to discuss with the Platform Team ways to optimize your application within your current quota. We usually don't approve quota requests without trying to make the existing quota work first. The more details you provide in your initial request, the shorter this conversation is likely to be!
+
+If you need help answering this question [check these examples](#examples-of-answers).
+
+### Step 5: Submit edit request changes 
+
+The Platform Services team needs to know if your application is using the current quotas efficiently. 
+
+* Once all the information has been compiled from [**Step 4**](#step-4-gather-information-for-your-request) click on the `Submit edit request` button
+
+* A pop-up will appear to allow space to include all the information you have collected and gathered, this place only allows for text information if you need to submit photos or files please do so in the next step as it is necessary and part of the request process
+
+### Step 6: Email the Platform Services team and book a meeting with them
+
+The last step is to email the request to the Platform Services team at [PlatformServicesTeam@gov.bc.ca](mailto:PlatformServicesTeam@gov.bc.ca) and include the following information: 
+
+* Attach screenshot of Sysdig dashboard (if available)
+* Attach additional images or files that supplement the already submitted information in the previous step
+* Book a meeting with the Platform Services team
```

Comment:
We don't typically require a meeting for every quota request as a rule. I think we should remove this line - Billy will request a meeting if he needs to follow up in a discussion.